src/rng/backends/rocrand/CMakeLists.txt

```diff
@@ -54,7 +54,9 @@
 
 set(LIB_NAME onemkl_rng_rocrand)
 set(LIB_OBJ ${LIB_NAME}_obj)
-find_package(rocRAND REQUIRED)
+find_package(hip REQUIRED)
+find_package(rocrand REQUIRED)
```

Comment:
Two naive questions:
1. Why do we find `hip` additionally here but not for `rocblas` and `rocsolver`?
2. Should we also fix a version for `rocrand` or drop the fixed versions for `rocblas` and `rocsolver`?

Reply:
Thank you for your review! I think both of these things need changing.

1. It looks like this was an oversight on my part. In the original scripts, rocsolver also links against `hip::host`, and rocblas uses its include directory. Presumably this still worked because the roc* packages depend on HIP, but my current solution is stictly incorrect. I'll fix this.
2. I think I should drop the version restrictions. I didn't have a fixed rocrand version because AMD's release notes didn't match the version of rocRAND that was installed on the system I was using - I had no idea what was the right.

Reply:
Done in [8e04fce](https://github.com/oneapi-src/oneMKL/pull/490/commits/8e04fce1724a382d3d80e9642bcfe6cdaca3dee8)